app/Repositories/UserRepository.php

```diff
@@ -41,4 +41,43 @@ public function create($username, $email, $password)
 
     }
 
+    /**
+     * Updates a user on the panel. Returns true if the update was successful.
+     *
+     * @param  string $username
+     * @param  string $email
+     * @param  string $password An unhashed version of the user's password.
+     * @return integer
+     */
+    public function update($id, $email, $isRoot, $password = null)
+    {
+        $user = User::findOrFail($id);
+
+        $user->email = $email;
+        $user->root_admin = $isRoot;
+
+        if(!empty($password)) {
+            $user->password = Hash::make($password);
+        }
+
+        $user->save();
+
+        return true;
+
+    }
+
+    /**
+     * Deletes a user on the panel. Returns true if the deletion was successful.
+     *
+     * @param  string $username
+     * @param  string $email
+     * @param  string $password An unhashed version of the user's password.
+     * @return integer
+     */
+    public function delete($id)
+    {
+        User::findOrFail($id)->delete();
+        return true;
```

Comment:
This return seems un-necessary, since `findOrFail()` will throw a `Illuminate\Database\Eloquent\ModelNotFoundException` if it fails to find the user. We don't even really need this function in the repository I would say, since it can all be accomplished directly from the `Models\User` action.